### DISCUSSION TITLE
1.1.30

### DISCUSSION THREAD

**Discussion by kirb** [wstyres/Zebra#2015] [Releases]
This is a bugfix release, as a follow-up to [1.1.29](https://github.com/zbrateam/Zebra/releases/tag/v1.1.29). Refer to that release for the major recent changes.

* Fixes packages failing to install when they use the “finish” file descriptor, also known as the `$CYDIA` variable, to signal to the package manager that the package requires a respring, reboot, etc.
* Zebra now supports search queries of only one character in some languages.  
  For performance reasons, Zebra has always required at least two characters to be typed before it performs a search. Searching for the letter “A”, for instance, can easily return thousands of results! That’s difficult for Zebra to process without creating frustrating lag as you’re typing your search query. However, this optimisation makes less sense for languages that don’t use the Latin alphabet, such as Chinese, Japanese, and Korean, and we’ve heard feedback that this makes it difficult to perform searches in those languages. Therefore, single-character searches are now supported in those languages.
* Fixes a crash when tapping “Add to Photos” after long-pressing an image in a package description.
* Fixes the App Icon settings screen not displaying any of the app icons on the rootless build of Zebra.
* Fixes deb files disappearing from their original location when opened in Zebra (e.g. from the Files app).

<hr /><em>This discussion was created from the release <a href='https://github.com/zbrateam/Zebra/releases/tag/v1.1.30'>1.1.30</a>.</em>

**Comment by augustobob** [comment]
Hey man, sorry for bother but there’s anyway I can make zebra work on XinaA15 jailbreak? I’m missing a good package manager 
Thanks
